ubuntudesign/developer.ubuntu.com
# Update HTML to bring inline with latest docs theme

## Done

Update HTML classes to bring in line with the latest docs theme

## QA

- Run ./run
- Go to http://127.0.0.1:8015/core and make sure it’s not as broken as http://developer.ubuntu.com-v1-dev.demo.haus/core
- The header and nav (including small/mobile) still need work to match the design, this will be done in separate branches
